packages/contentful--app-scripts/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "exclude": ["node_modules", "**/*.test.ts"],
+  "compilerOptions": {
+    "outDir": "./lib"
```

Comment:
Can we enable `declaration`, so people have types when importing from this package? This would also require a `types` prop in package.json.